### PR TITLE
TM-6152 search on remarks

### PR DESCRIPTION
**Pull request by fimbriano**
Panel Meetings
- Added a remarks filter drop down
-  Added remark pills to each panel meeting item
-- Currently it shows a hard coded set of remarks, so you can see/test the functionality until the api is implemented

[Ticket](https://metaphase.atlassian.net/browse/TM-6152)

### CODE REVIEW

File: src/sass/_panelRemarks.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.panel-remarks-container {
```
Comment (jlam-mpc) [inline src/sass/_panelRemarks.scss:1]: I'd want to suggest figuring out a way to reuse some of the styles in _remarks.scss and adding some additional parent classnames to differentiate anything you want

Not sure if that made sense (can explain further if you want to chat) but it's just to reduce stylesheet bloating, repeating too many lines of styles that overlap, and also if we change the themes of layout remarks pills in the future it'd be slightly easier to tackle it in one location

Comment (jlam-mpc) [reply src/sass/_panelRemarks.scss:1]: So you can probably remove this stylesheet and add it to remarks.scss

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx [hunk 2]
```diff
@@ -145,16 +154,18 @@ const PanelMeetingSearch = ({ isCDO }) => {
...
-    const getSelected = item => !!selected.find(f => f.code === item.code);
-    return items.map(item =>
-      (<ListItem
-        key={item.code}
+    const codeOrSeqNum = has(items[0], 'seq_num') ? 'seq_num' : 'code';
+    const getSelected = item => !!selected.find(f => f[codeOrSeqNum] === item[codeOrSeqNum]);
+
+    return items.map(item => (
+      <ListItem
+        key={codeOrSeqNum === 'code' ? item.code : item.codeOrSeqNum}
```
Comment (jlam-mpc) [inline src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:162]:
```suggestion
        key={codeOrSeqNum === 'code' ? item.code : item[codeOrSeqNum]}
```

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx [hunk 3]
```diff
@@ -145,16 +154,18 @@ const PanelMeetingSearch = ({ isCDO }) => {
   };
 
   const renderSelectionList = ({ items, selected, ...rest }) => {
```
Comment (jlam-mpc) [inline src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:156]: Actually just remembered we have a utility for renderSelectionList in utilities.js that we should replace with in this file.

you can add a conditional to that function to check for seq_num and use that instead

**Review by jlam-mpc — CHANGES_REQUESTED** [review]
Really clean and satisfies the requirements! Just had a few questions and suggestions

File: src/Components/Panel/PanelMeetingSearchRow/PanelMeetingSearchRow.jsx [hunk 4]
```diff
@@ -16,29 +19,51 @@ const showEditPanelMeeting = usePanelAdmin() && usePanelAdminPanelMeeting();
 
 const PanelMeetingSearchRow = ({ isCDO, pm }) => {
   const pmSeqNum = get(pm, 'pm_seq_num') || FALLBACK;
+  const remarks = get(pm, 'remarks') || [{ text: '2nd Tour JO', rc_code: 'A' }, { text: 'ATA 03/03/2012', rc_code: 'T' }, { text: 'Approved off-panel on 03/03/2012', rc_code: 'M' }, { text: 'BOP', rc_code: 'E' }, { text: '123 BSAC', rc_code: 'B' }, { text: 'custom remark here', rc_code: 'M' }, { text: 'Breaks/curtails 2012 PSP/SIP link', rc_code: 'M' }, { text: 'CA EL LDP', rc_code: 'P' }, { text: '123 CA LNA Class', rc_code: 'B' }];
```
Comment (fimbriano) [inline src/Components/Panel/PanelMeetingSearchRow/PanelMeetingSearchRow.jsx:22]:
```suggestion
  // TODO: replace fallback with [], once api portion is complete
  const remarks = get(pm, 'remarks') || [{ text: '2nd Tour JO', rc_code: 'A' }, { text: 'ATA 03/03/2012', rc_code: 'T' }, { text: 'Approved off-panel on 03/03/2012', rc_code: 'M' }, { text: 'BOP', rc_code: 'E' }, { text: '123 BSAC', rc_code: 'B' }, { text: 'custom remark here', rc_code: 'M' }, { text: 'Breaks/curtails 2012 PSP/SIP link', rc_code: 'M' }, { text: 'CA EL LDP', rc_code: 'P' }, { text: '123 CA LNA Class', rc_code: 'B' }];
```

File: src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx (hunk 2 above)
Comment (fimbriano) [reply src/Components/Panel/PanelMeetingSearch/PanelMeetingSearch.jsx:162]: it was actually item.seq_num haha good catch!

**Review by jlam-mpc — APPROVED** [review]

**Pull request merged**